### ISSUE TITLE
A easier way to add devices and use ROS to control (e.x. UR5e and a gripper)

### ISSUE THREAD

**Issue by ipa-rwu** [cyberbotics/webots#2064]
Hello,

As I mentioned before  I try to add a gripper on UR5e and control through ROS. 

I created some ugly code. This is the repository for [webots world](https://github.com/ipa-rwu/ur5_2f_webots). And this is the repository for [ROS control](https://github.com/ipa-rwu/ur5e_robotiq). 
Now it works for **webots 2020a**. I tried it on 2020b version. It seems that there are some errors in the proto of "robotiq 2finger gripper".

You can use ``collision.web`` as a world. Choosing "modify" as "controller" for UR5e. 
And then `roslaunch ur5e_robotiq_gazebo ur5e_robotiq_webots.launch`. I hope you are able to control the arm and gripper in webots now.

I also added two touch sensors on red boxes. Actually I want to add a touch sensor on the gripper as mentioned  #1849 (I am sorry that I had no time to answer quickly). I use the same way to add a touch sensor on the gripper as on the box.  But the gripper treated it as an obstacle. 

Basically I created "[gripper_trajectory_follower_2joints.py](https://github.com/ipa-rwu/ur5_2f_webots/blob/master/controllers/modify/gripper_trajectory_follower_2joints.py)" . But some part of the code is reusable from "[trajectory_follower.py](https://github.com/ipa-rwu/ur5_2f_webots/blob/master/controllers/modify/trajectory_follower.py)" And I also added a joint of the gripper in [joint_state_publisher.py](https://github.com/ipa-rwu/ur5_2f_webots/blob/master/controllers/modify/joint_state_publisher.py).

Maybe it is possible to simplify adding new devices with ROS control.

**Comment by omichel** [comment]
I am closing this stalled issue. Feel free to re-open if needed.
